internal/constants/scripts.go

```diff
@@ -186,13 +179,28 @@ find_host_directory_by_mac_address
 // Make sure we do not run any of the following functions if there was no matching config.
 if [ "$host_dir" ]
   then
-    # remove default connection file create by NM(nm-initrd-generator). This is a WA until
-    # NM is back to supporting priority between nmconnections
-    rm -f ${ETC_NETWORK_MANAGER}/*
-    echo "Removing default connection files in '$ETC_NETWORK_MANAGER'"
     set_logical_nic_mac_mapping
     update_interface_names_by_mapping_file
     copy_nmconnection_files_to_nm_config_dir
 fi
 echo "PreNetworkConfig End"
 `
+
+const PreNetworkConfigService = `
```

Comment:
If this is only being used for minimal iso can we name it as such?

Maybe `MinimalISONetworkConfigService` or something?

Reply:
Otherwise it's easy to confuse with `templates/pre-network-manager-config.service`

Reply:
Done.